config.go

```diff
@@ -976,7 +976,7 @@ func extractBitcoindRPCParams(bitcoindConfigPath string) (string, string, string
 	}
 
 	chainDir := "/"
-	netRE, err := regexp.Compile(`(?m)^\s*(testnet|regtest)=[\d]+`)
+	netRE, err := regexp.Compile(`(?m)^\s*(testnet|regtest)\s*=\s*[\d]+`)
```

Comment:
Please remove this when rebasing as we're now using `lnd` config instead of `bitcoind` config to detect the correct network.